### DISCUSSION TITLE
LoTW is not synchronizing

### DISCUSSION THREAD

**Discussion by 4O4A** [magicbug/Cloudlog#1434] [Q&A]
Hi all,

I created cron job for LoTW as per instructions, and upload works flawlesly. However, the opposite direction is not working with me, confirmations are not synchronized. It can be synchronized manually, though. 
What am I missing?

Regards,
Dragan, 4O4A

**Comment by magicbug** [comment]
If its not downloading then its likely login info for LoTW that's incorrect if you manually run the URL in browser window do you get any errors? have you added login into for Lotw to your user profile

**Comment by 4O4A** [comment]
No errors, manual syncing works fine, I guess. Upload works fine, too.
After I execute manual sync, in the same window I got a list of fetched QSLs with basic info (date, mode, date of QSO, time of QSLs are 00:00). That is all I see on screen. To get back to log view, I have to hit "back" in browser, and I can see that LoTW RCVD arrows are green as they should be for given QSOs.

**Reply by 4O4A** [reply]
This is a screenshot of what I see after Manual Sync:
https://www.4o4a.com/LoTW-fetch.jpg

**Reply by magicbug** [reply]
Surely if the  LoTW RCVD and sent arrows are green its working and there for a match?

**Reply by magicbug** [reply]
Looks like this is a issue with the cron in that case perhaps Copy and paste cronline

**Reply by 4O4A** [reply]
Sure, here it is -- uploads QSOs hourly, as expected:
curl --silent https://leo.4o4a.com/lotw/lotw_upload &>/dev/null
